pennylane/templates/layers/simplified_two_design.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Contains the ``SimplifiedTwoDesign`` template.
+"""
+# pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
+from pennylane import numpy as np
+from pennylane.templates.decorator import template
+from pennylane.ops import CZ, RY
+from pennylane.templates import broadcast
+from pennylane.templates.utils import (
+    _check_shape,
+    _check_wires,
+    _check_number_of_layers,
+    _check_type,
+    _get_shape,
+)
+
+
+@template
+def entangler(par1, par2, wires):
+    """Implements a two qubit unitary consisting of a controlled-Z entangler and Pauli-Y rotations.
+
+    Args:
+         par1 (float or qml.Variable): parameter of first Pauli-Y rotation
+         par2 (float or qml.Variable): parameter of second Pauli-Y rotation
+         wires (list): two wire indices that unitary acts on
+    """
+
+    CZ(wires=wires)
+    RY(par1, wires=wires[0])
+    RY(par2, wires=wires[1])
+
+
+@template
+def SimplifiedTwoDesign(initial_layer, weights, wires):
```

Comment:
The aim would be to make it apparent that here the parameters are accepted instead of the operations themselves (e.g. might be interpreted in a way that one has to give some sort of a generic layer structure).
```suggestion
def SimplifiedTwoDesign(initial_layer_weights, weights, wires):
```

Reply:
Just that this name is so long...But I just can't think of a better name, so I changed it to your one...